sdks/java/extensions/python/build.gradle

```diff
@@ -21,19 +21,16 @@ applyJavaNature( automaticModuleName: 'org.apache.beam.sdk.extensions.python')
 
 description = "Apache Beam :: SDKs :: Java :: Extensions :: Python"
 
-evaluationDependsOn(":runners:core-construction-java")
-
 dependencies {
     implementation library.java.vendored_guava_32_1_2_jre
     implementation library.java.vendored_grpc_1_60_1
     implementation library.java.slf4j_api
     implementation project(path: ":model:pipeline", configuration: "shadow")
-    implementation project(path: ":runners:core-construction-java")
     implementation project(path: ":sdks:java:core", configuration: "shadow")
-    implementation project(path: ":sdks:java:transform-service:launcher")
+    implementation project(path: ":sdks:java:transform-service:launcher") // not "shadow" configuration, which is an uberjar
```

Comment:
Ditto regarding the inconsistency when depending with other projects vs this one.